routing/routing_session.hpp

```diff
@@ -35,7 +38,19 @@ struct SpeedCameraRestriction
   SpeedCameraRestriction() : m_index(0), m_maxSpeedKmH(numeric_limits<uint8_t>::max()) {}
 };
 
-class RoutingSession
+class TrafficCache
```

Comment:
Moreover, you don't need TrafficInfoGetterNoJam. Just don't call Set(), and it will behave like your current implementation of TrafficInfoGetterNoJam.